testing/kuttl/e2e-other/exporter-upgrade/README.md

```diff
@@ -0,0 +1,15 @@
+The exporter-upgrade test makes sure that PGO updates two extensions used for monitoring.
+This avoids an error where a user might update to a new PG image with a newer extension,
+but with an older extension operative.
+
+00-01
+This starts up a cluster with a purposely outdated extension. Because we want a particular
+extension, the image used here is hard-coded and not publicly available.
+
+02-03
+The cluster is updated, again with a hardcoded image. Due to the change made in
+https://github.com/CrunchyData/postgres-operator/pull/3400, this should no longer produce errors.
+However, what has been observed is that there are occasionally a few errors logged
+before the extension is updated. So this checks that there are no more than 2 errors, since that
+was the observed number during manual tests of the check; and since without the fix,
+there were more errors observed.
```

Comment:
I do not understand. What are the 2 errors that we expect to find?

Reply:
Updated